app/[locale]/(interview)/layout.tsx

```diff
@@ -0,0 +1,16 @@
+export const metadata = {
+  title: 'Interview',
+  description: 'Network Canvas Interview',
+};
+
+export default function InterviewLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <html lang="en">
+      <body className="flex h-screen bg-navy-taupe text-white">{children}</body>
```

Comment:
This className is giving some warning in the console, you might wanna check this

Reply:
resolved here [58911cc](https://github.com/complexdatacollective/Studio/pull/30/commits/58911cc5171feb54ca4e4768b0e3a1a7d5367d53)